react/src/containers/App.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react'
+
+import Show from '../components/Show'
+
+
+class App extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      shows
```

Comment:
Are you trying to assign `shows` to a null value? Or is `shows` coming in from somewhere? Just be sure to initialize your variable properly